lib/net/pool.js

```diff
@@ -213,7 +222,12 @@ class Pool extends EventEmitter {
     this.headerNext = null;
 
     const tip = this.chain.tip;
-
+    if (this.options.neutrino) {
+      this.headerChain.push(new HeaderEntry(tip.hash, tip.height));
+      this.cfHeaderChain = new List();
```

Comment:
If we really need this as a class property it should be defined in the constructor